### PR TITLE
Update/lib security checkup test

### PR DESCRIPTION
**Pull request by jhnstn**
part of #3942

### CODE REVIEW

File: client/lib/security-checkup/test/actions.js [hunk 1]
```diff
@@ -1,41 +1,48 @@
...
-const chai = require( 'chai' ),
-	expect = chai.expect,
-	sinon = require( 'sinon' ),
-	sinonChai = require( 'sinon-chai' );
+
+import chai, { expect } from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
 
 chai.use( sinonChai );
```
Comment (blowery) [inline client/lib/security-checkup/test/actions.js]: drop this bit

File: client/lib/security-checkup/test/actions.js [hunk 2]
```diff
@@ -1,41 +1,48 @@
...
+import testConstants from './fixtures/constants';
+
+import Dispatcher from 'dispatcher';
+import analytics from 'lib/analytics';
+import undocumentedMe from 'lib/wpcom-undocumented/lib/me';
+
+import { actions } from '../constants';
+import SecurityCheckupActions from '../actions';
 
 const sandbox = sinon.sandbox.create();
```
Comment (blowery) [inline client/lib/security-checkup/test/actions.js]: should use useSandbox in `test/helpers/use-sinon`

**Comment by blowery** [comment]
fixed, rebased, forced

**Comment by blowery** [comment]
cc @gziolo

File: client/lib/security-checkup/test/actions.js [hunk 3]
```diff
@@ -3,34 +3,37 @@
...
-import analytics from 'lib/analytics';
 import undocumentedMe from 'lib/wpcom-undocumented/lib/me';
 
-import { actions } from '../constants';
-import SecurityCheckupActions from '../actions';
+describe( 'SecurityCheckupActions', () => {
+	let sandbox, actions, SecurityCheckupActions, testConstants, analytics;
 
-const sandbox = sinon.sandbox.create();
+	useSandbox( sandy => {
```
Comment (gziolo) [inline client/lib/security-checkup/test/actions.js]: Minor, it could be one-liner ;)

**Comment by gziolo** [comment]
It works as expected. Should be good to go 👍

**Pull request merged**